interactions/api/models/user.py

```diff
@@ -96,3 +99,15 @@ def banner_url(self) -> Optional[str]:
         url = f"https://cdn.discordapp.com/banners/{int(self.id)}/{self.banner}"
         url += ".gif" if self.banner.startswith("a_") else ".png"
         return url
+
+    @property
+    def presence(self) -> "Presence":
+        """
+        Returns the presence of the user.
+
+        :return: Presence of the user (None will be returned if not cached)
+        :rtype: Presence
+        """
+        from .gw import Presence
+
+        return self._client.cache[Presence].get(self.id)
```

Comment:
If `None` can be returned, make the return type and the typehint `Optional`?

Reply:
Yeah you're right. I did add this